datafusion/core/src/datasource/physical_plan/parquet/metrics.rs

```diff
@@ -29,8 +29,10 @@ use crate::physical_plan::metrics::{
 pub struct ParquetFileMetrics {
     /// Number of times the predicate could not be evaluated
     pub predicate_evaluation_errors: Count,
-    /// Number of row groups pruned using
-    pub row_groups_pruned: Count,
+    /// Number of row groups pruned by bloom filters
+    pub row_groups_pruned_sbbf: Count,
```

Comment:
Is "sbbf" a commonly known acronym?
Or could we instead call it "row_groups_pruned_bloom_filter"?

Reply:
> Is "sbbf" a commonly known acronym? 

No I don't think it is commonly known -- I definitely didn't know about it until I started working on parquet (BTW it stands for [Split Block Bloom Filter](https://github.com/apache/parquet-format/blob/master/BloomFilter.md#technical-approach))

> Or could we instead call it "row_groups_pruned_bloom_filter"?

I think that is a good idea